### PR TITLE
PG-954 Telemetry (16)

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   docs/telemetry.md

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -1,53 +1,386 @@
-# Telemetry on Percona Distribution for PostgreSQL
+# Telemetry and data collection
 
-Percona telemetry fills in the gaps in our understanding of how you use Percona Distribution for PostgreSQL to improve our products. Participation in this anonymous program is optional. You can opt-out if you prefer to not share this information.
+Percona collects usage data to improve its software. The telemetry feature helps us identify popular features, detect problems, and plan future improvements.
```
Comment (janwieremjewicz) [inline docs/telemetry.md]: Can we maybe mention here already that we do not pull any sensitive data, data is basically anonymized

File: docs/telemetry.md [hunk 2]
```diff
@@ -1,53 +1,386 @@
...
+* The information about the operating system such as name, architecture, the list of Percona packages. See more in the [Telemetry Agent section](#telemetry-agent).
+* The metrics from the database instance. See more in the [percona_pg_telemetry section](#percona_pg_telemetry).
+
+## What is NOT collected
+
+Percona protects your privacy and doesn't collect any personal information about you like database names, user names or credentials or any user-entered values. 
+
+All collected data is anonymous, meaning it can't be traced back to any individual user. To learn more about how Percona handles your data, read the [Percona Privacy statement](https://www.percona.com/privacy-policy).
+
+You control whether to share this information. Participation in this program is completely voluntary. If don't want to share anonymous data, you can [disable telemetry](#disable-telemetry).
```
Comment (janwieremjewicz) [inline docs/telemetry.md]: Seems like a great place to put a message that open source needs feedback even more than closed sourced solutions

File: docs/telemetry.md [hunk 3]
```diff
@@ -1,53 +1,386 @@
...
+|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| See how people use your software | Telemetry collects anonymous data on how users interact with our software. This tells developers which features are popular, which ones are confusing, and if anything is causing crashes. |
+| Identify issues early       | Telemetry can catch bugs or performance problems before they become widespread. |
+
+Benefits for users in the long run:
+
+| Advantages                | Description                                                                                                         |
+|---------------------------|---------------------------------------------------------------------------------------------------------------------|
+| Faster bug fixes          | With telemetry data, developers can pinpoint issues affecting specific users and prioritize fixing them quickly.  |
```
Comment (janwieremjewicz) [inline docs/telemetry.md]: specific users suggests that users can be recognized. Can we rephrase this?

File: docs/telemetry.md [hunk 4]
```diff
@@ -1,53 +1,386 @@
...
+
+    * Collects OS-level metrics
+
+    * Reads the Metrics File, adds the OS-level metrics
+
+    * Sends the full set of metrics to the Percona Platform
+
+    * Collects the list of installed Percona packages using the local package manager
+
+The telemetry also uses the Percona Platform with the following components:
```
Comment (janwieremjewicz) [inline docs/telemetry.md:60]: I am not sure on the status of a product called Percona Platform but it seems that this mention here refers to it while I am almost certain it does not

File: docs/telemetry.md [hunk 5]
```diff
@@ -1,53 +1,386 @@
...
+
+* Telemetry Service - offers an API endpoint for sending telemetry. The service handles incoming requests. This service saves the data into Telemetry Storage.
+
+* Telemetry Storage - stores all telemetry data for the long term.
+
+### `percona_pg_telemetry`
+
+`percona_pg_telemetry` is an extension to collect telemetry data in PostgreSQL. It is added to Percona Distribution for PostgreSQL and is automatically loaded when you install a PostgreSQL server.
+
+`percona_pg_telemetry` collects metrics from the database instance daily to the Metrics File. It creates a new Metrics File for each collection. Before generating a new file, the `percona_pg_telemetry` deletes the Metrics Files that are older than seven days. This process ensures that only the most recent week's data is maintained.
```
Comment (janwieremjewicz) [inline docs/telemetry.md]: good place to mention that such file can be inspected to see what data is collected

**Review by janwieremjewicz — APPROVED** [review]
There are no blockers to this PR though I would appreciate addressing the comments I made first

File: docs/telemetry.md (hunk 1 above)
Comment (nastena1606) [reply docs/telemetry.md]: Added

File: docs/telemetry.md (hunk 3 above)
Comment (nastena1606) [reply docs/telemetry.md]: Rephrased

File: docs/telemetry.md (hunk 5 above)
Comment (nastena1606) [reply docs/telemetry.md]: Added

**Review by Naeem-Akhter — APPROVED** [review]
Looks good to me!

**Review by artemgavrilov — APPROVED** [review]

**Pull request merged**